crates/eframe/src/native/epi_integration.rs

```diff
@@ -75,7 +75,7 @@ pub fn read_window_info(
 pub fn window_builder<E>(
     event_loop: &EventLoopWindowTarget<E>,
     title: &str,
-    native_options: &epi::NativeOptions,
+    native_options: &mut epi::NativeOptions,
```

Comment:
Adding `mut` everywhere is quite ugly.

This can be fixed by using `Rc<dyn FnOnce(…)>` as the callback type, instead of just `Box`. That will also allow cloning.